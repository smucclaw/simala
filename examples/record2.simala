a = { x = 2, y = 3 };
b = { x = 'atom, z = 3 };
#eval merge(a, b)
