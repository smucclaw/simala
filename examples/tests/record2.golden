{x = 'atom,y = 3,z = 3}
